### ISSUE TITLE
Alternative project for PlantUML?

### ISSUE THREAD

**Issue by aadrian** [plantuml/smetana#1]
What about? https://github.com/nidi3/graphviz-java

Would it work for PlantUML? *(Since it seems to cover quite allot of dot)*

**Comment by arnaudroques** [comment]
Yes, https://github.com/nidi3/graphviz-java would work for java.

Actually, it does already work! See http://plantuml.com/vizjs

We did not integrate graphviz-java as-it, but we did follow the same strategy : running vis.js using J2V8

**Comment by aadrian** [comment]
@arnaudroques Thank you for the explanation. I haven't seen that before, maybe because
http://plantuml.com/sitemap seems to be broken since it's not showing links to **all** site pages :).
